src/whylogs/core/statistics/constraints.py

```diff
@@ -213,7 +232,27 @@ def __init__(
         self.value = value
         self.upper_value = upper_value
 
-        if self.op == Op.BTWN:
+        if self.op in (Op.IN_SET, Op.CONTAIN_SET, Op.EQ_SET):
+            if value is not None or upper_value is not None or second_field is not None or third_field is not None or reference_set is None:
```

Comment:
`if any([value, upper_value, second_field, third_field, reference_set]):`

Reply:
Thanks, so much better, but shouldn't it be: `if any([value, upper_value, second_field, third_field, not reference_set])` ?